### PR TITLE
Added new GitHub action to schedule blog post merges

### PR DESCRIPTION
**Pull request by flippedcoder**
We want to get to a point where a blog post PR that is approved can be scheduled to auto-merge on a specified date.

Here is [the doc I used to make this YAML file](https://github.com/marketplace/actions/merge-schedule) and it explains all of the functionality.

### CODE REVIEW

**Review by julieg18 — APPROVED** [review]
Looks good!

File: .github/workflows/merge-schedule.yaml [hunk 1]
```diff
@@ -0,0 +1,24 @@
+name: Merge Scheduler
+on:
+  pull_request:
+    types:
+      - opened
+      - edited
+      - synchronize
+  schedule:
+    # https://crontab.guru/every-hour
+    - cron: 0 6 * * *
```
Comment (jorgeorpinel) [inline .github/workflows/merge-schedule.yaml:10]: That's not "at every hour". This is "at 6am" (every day).

Comment (jorgeorpinel) [reply .github/workflows/merge-schedule.yaml:10]: I'd just remove that comment to avoid confusion
```suggestion
    - cron: 0 6 * * *
```

Comment (flippedcoder) [reply .github/workflows/merge-schedule.yaml:10]: Yes, that's correct. We don't need to job to run every hour.

**Comment by flippedcoder** [comment]
@julieg18 Would you mind merging this for me? I don't have merge permissions in this repo yet.

File: .github/workflows/merge-schedule.yaml [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+    steps:
+      - uses: gr2m/merge-schedule-action@v1
+        with:
+          # Merge method to use. Possible values are merge, squash or
+          # rebase. Default is merge.
+          merge_method: squash
+          #  Time zone to use. Default is UTC.
+          time_zone: "America/Los_Angeles"
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```
Comment (shcheklein) [inline .github/workflows/merge-schedule.yaml:23]: it's a good practice to have a new line at the end of the file (thus you see this red icon)

**Comment by casperdcl** [comment]
Can we also have this documented in https://dvc.org/doc/user-guide/contributing/docs?

**Comment by jorgeorpinel** [comment]
Maybe in https://dvc.org/doc/user-guide/contributing/blog but this seems like an internal tool so not sure it's needed there.

**Comment by shcheklein** [comment]
Can it be part of the PR template (+ check list of things to do that DevRel team usually includes these days) 

https://github.com/iterative/dvc.org/blob/master/.github/PULL_REQUEST_TEMPLATE.md

can we create multiple templates?

**Pull request merged**